### PR TITLE
POJO Implementation of AbstractFSRollingOutputOperator

### PR DESCRIPTION
**Pull request by pm48**
An Implementation of AbstractFSRollingOutputOperator which takes any POJO as input, serializes the POJO as Hive delimiter separated values which are written in text files to hdfs.

### CODE REVIEW

File: contrib/src/main/java/com/datatorrent/contrib/hive/FSPojoToHiveOperator.java [hunk 1]
```diff
@@ -0,0 +1,241 @@
...
+        value = ((GetterInt<Object>)getters.get(index)).get(tuple);
+        break;
+      case LONG:
+        value = ((GetterLong<Object>)getters.get(index)).get(tuple);
+        break;
+      case FLOAT:
+        value = ((GetterFloat<Object>)getters.get(index)).get(tuple);
+        break;
+      case DOUBLE:
+        value = ((GetterDouble<Object>)getters.get(index)).get(tuple);
```
Comment (vrozov) [inline contrib/src/main/java/com/datatorrent/contrib/hive/FSPojoToHiveOperator.java]: Cases for boolean, byte, char, short, int, long, float and double are subject to unnecessary boxing, leading to GC and performance degradation. As far as I can see boxing can be avoided. Please stop by on Tuesday to discuss.

File: contrib/src/main/java/com/datatorrent/contrib/hive/FSPojoToHiveOperator.java [hunk 2]
```diff
@@ -0,0 +1,241 @@
...
+      case DATE:
+        value = ((Getter<Object,Date>)getters.get(index)).get(tuple);
+        break;
+      case TIMESTAMP:
+        value = ((Getter<Object,Timestamp>)getters.get(index)).get(tuple);
+        break;
+      case OTHER:
+        value = ((Getter<Object,Object>)getters.get(index)).get(tuple);
+        break;
+    }
```
Comment (vrozov) [inline contrib/src/main/java/com/datatorrent/contrib/hive/FSPojoToHiveOperator.java]: It is better to have default case and throw an exception or assert to make sure that if new types are introduced later, the corresponding case statement is added to the switch here.

File: contrib/src/main/java/com/datatorrent/contrib/hive/FSPojoToHiveOperator.java (hunk 1 above)
Comment (PramodSSImmaneni) [inline contrib/src/main/java/com/datatorrent/contrib/hive/FSPojoToHiveOperator.java]: Please correct this. The method getValue cause unnecessary boxing, try to get bytes directly from the primitive

**Comment by pm48** [comment]
Need to add more test cases for this operator.

**Pull request closed without merging**